CHANGELOG.md

```diff
@@ -1,32 +1,28 @@
 <a name="0.18.0"></a>
-# [0.18.0](https://github.com/redux-observable/redux-observable/compare/v0.17.0...v0.18.0) (2018-02-07)
 
+# [0.18.0](https://github.com/redux-observable/redux-observable/compare/v0.17.0...v0.18.0) (2018-02-07)
```

Comment:
We'll need to ignore this file since it is autogenerated and would always need reformatting for every release. https://prettier.io/docs/en/ignore.html

Reply:
Added to the ignore and amended :)